modules/uhsg_search/uhsg_search.module

```diff
@@ -55,7 +59,7 @@ function uhsg_search_article_is_degree_programme_specific($node) {
 /**
  * Implements hook_search_api_solr_query_alter().
  */
-function uhsg_search_search_api_solr_query_alter(\Solarium\QueryType\Select\Query\Query $solarium_query, \Drupal\search_api\Query\QueryInterface $query) {
+function uhsg_search_search_api_solr_query_alter(Query $solarium_query, QueryInterface $query) {
```

Comment:
These "with/without namespace" changes are a bit weird (sometimes the tool wants to use the full namespace, sometimes not). In core and contrib module api definitions the full namespace is typically used: `function hook_search_api_solr_query_alter(\Solarium\Core\Query\QueryInterface $solarium_query, \Drupal\search_api\Query\QueryInterface $query) {`.